ompi/mca/coll/libnbc/coll_libnbc_component.c

```diff
@@ -97,7 +98,6 @@ libnbc_open(void)
                               NULL);
     if (OMPI_SUCCESS != ret) return ret;
 
-    OBJ_CONSTRUCT(&mca_coll_libnbc_component.active_requests, opal_list_t);
```

Comment:
Would it be terrible if you split this out into a separate commit?  Can be on this same PR, but just logically, it has nothing to do with this commit (and the commit message you provided).

Thanks!